### ISSUE TITLE
Hey how can we access admin

### ISSUE THREAD

**Issue by rajivchaulagain** [devat-youtuber/mern-ecommerce#11]

**Comment by AbhayChandel13** [comment]
yes, Same Issue  ! can you please give access to admin panel   also ,only then it is useful

**Comment by azadpunchy** [comment]
you need to link your own mongodb database
then open your mongodb where your users are stored after signup
then change the role of user from 0 to 1 
Now your user is Admin
